data/data/azure/master/master.tf

```diff
@@ -0,0 +1,102 @@
+resource "azurerm_network_interface" "master" {
+  count               = "${var.instance_count}"
+  name                = "${var.cluster_id}-master-nic-${count.index}"
+  location            = "${var.region}"
+  resource_group_name = "${var.resource_group_name}"
+
+  ip_configuration {
+    subnet_id                     = "${var.subnet_id}"
+    name                          = "master-${count.index}"
+    private_ip_address_allocation = "Dynamic"
+  }
+}
+
+resource "azurerm_network_interface_backend_address_pool_association" "master" {
+  count                   = "${var.instance_count}"
+  network_interface_id    = "${element(azurerm_network_interface.master.*.id, count.index)}"
+  backend_address_pool_id = "${var.elb_backend_pool_id}"
+  ip_configuration_name   = "master-${count.index}"                                          #must be the same as nic's ip configuration name.
+}
+
+resource "azurerm_network_interface_backend_address_pool_association" "master_internal" {
+  count                   = "${var.instance_count}"
+  network_interface_id    = "${element(azurerm_network_interface.master.*.id, count.index)}"
+  backend_address_pool_id = "${var.ilb_backend_pool_id}"
+  ip_configuration_name   = "master-${count.index}"                                          #must be the same as nic's ip configuration name.
+}
+
+resource "azurerm_lb_nat_rule" "master" {
```

Comment:
can we have information on why this specific resource is required? If this is for debugging purpose mark todo to remove..

Reply:
for debugging purpose, removing

Reply:
removed in https://github.com/openshift/installer/pull/1454/commits/d02f9bfcb2de03a3416a1f6e909cef85d19dc8ce